src/db/types.ts

```diff
@@ -77,6 +77,8 @@ export type BlockDownloadedVideo = BlockPrototype & { type: "downloaded_video";
 
 export type Block = BlockNote | BlockYoutubeInput | BlockDownloadedVideo;
 
+export const isBlock = (doc: DbDocument): doc is Block => doc.document_type === "block";
```

Comment:
note to @tlmn : this is a useful way when you are doing type switches.  Typescript will identify anything passed through this function as a `Block`